<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthWindow</class>
 <widget class="QDialog" name="AuthWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>160</width>
     <height>246</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="loginInput">
        <property name="placeholderText">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordInput">
        <property name="placeholderText">
         <string>Password</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="authEnterButton">
        <property name="text">
         <string>Войти</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="regEnterButton">
        <property name="text">
         <string>Зарегистрироваться</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="authLabel">
        <property name="text">
         <string>Еще нет аккаунта?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="regLabel">
        <property name="text">
         <string>Уже есть аккаунт?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="authRegButton">
        <property name="text">
         <string>Регистрация</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="regLoginButton">
        <property name="text">
         <string>Войти</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="passLoginFormat">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
